src/dataregistry/db_basic.py

```diff
@@ -173,13 +169,25 @@ def __init__(self, db_connection, get_db_version=True):
         # Load all existing tables
         self._metadata.reflect(self._engine, db_connection.schema)
 
-        # Fetch and save db versioning if present and requested
+        # Fetch and save db versioning, assoc. production schema
+        # if present and requested
+        self._prod_schema = None
         if db_connection.dialect == "sqlite":
             prov_name = "provenance"
         else:
             prov_name = ".".join([self._schema, "provenance"])
+
+            # prov_table = self._metadata.tables[prov_name]
         if prov_name in self._metadata.tables and get_db_version:
```

Comment:
More general comment, is there any reason why `prov_name` shouldn't in the list of tables, i.e.,

`if prov_name in self._metadata.tables`

If this is not true, shouldnt an error be raised?

Reply:
I guess so.  This is not code I changed. Perhaps the thinking was then that there should be support for old-style databases with no provenance table, but there is no reason for it now.